src/webview/leetCodeResultProvider.ts

```diff
@@ -2,29 +2,35 @@
 // Licensed under the MIT license.
 
 import { Disposable, ExtensionContext, ViewColumn, WebviewPanel, window } from "vscode";
+import { leetCodeChannel } from "../leetCodeChannel";
+import { MarkdownEngine } from "./MarkdownEngine";
 
 class LeetCodeResultProvider implements Disposable {
 
     private context: ExtensionContext;
     private panel: WebviewPanel | undefined;
+    private mdEngine: MarkdownEngine;
 
     public initialize(context: ExtensionContext): void {
+        this.mdEngine = new MarkdownEngine();
```

Comment:
Having a question here. I believe that we have fully decoupled the markdown engine with the webviews. So do we still need to create new markdown instance in each webview?